src/transform/ThemeTransform.js

```diff
@@ -41,11 +41,18 @@ const setTheme = (document, theme) => {
  * https://en.wikipedia.org/wiki/Template:Football_kit/pattern_list
  * @type {RegExp}
  */
-const footballTemplateImageFilenameRegex =
-  new RegExp('Kit_(body|socks|shorts|right_arm|left_arm)(.*).png$')
+const footballTemplateImageFilenameRegex = /Kit_(body|socks|shorts|right_arm|left_arm)(.*).png$/
+
+/**
+ * Dialog image filename regex.
+ * https://commons.wikimedia.org/wiki/Comparison_of_icon_sets
+ * @type {RegExp}
+ */
+const dialogImageFilenameRegex = /-[Dd]ialog-\w+\.svg(\.png)?$/
```

Comment:
You might want to make this whole regex case insensitive like `/-dialog-\w+\.svg(\.png)?$/i`

Reply:
Done